.github/workflows/rust.yml

```diff
@@ -8,15 +8,91 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  # Minimum supported Rust version (MSRV)
+  ACTION_MSRV_TOOLCHAIN: 1.43.0
```

Comment:
Note: this matches the (current) latest version at https://catalog.redhat.com/software/containers/rhel8/rust-toolset/5b9c6aa85a134643ef2ecc63.